frontend/components/admin/allowance.vue

```diff
@@ -1,35 +1,36 @@
 <template>
   <div>
     <v-spacer></v-spacer>
-    <v-dialog v-model="dialog" max-width="500px">
-      <!-- <template v-slot:activator="{ on }">
-        <v-btn color="primary" dark class="mb-2" @click="newItem()">New Item</v-btn>
-      </template> -->
+    <v-dialog v-model="dialog" persistent max-width="500px">
       <v-card>
-        <v-card-title>
-          <span class="headline">{{ formTitle }}</span>
-        </v-card-title>
-
+        <v-toolbar flat color="white">
+          <v-toolbar-title>Allowance Configs</v-toolbar-title>
+        </v-toolbar>
+        <v-divider></v-divider>
         <v-card-text>
-          <v-container grid-list-md>
-            <v-layout wrap>
-              <v-flex xs12>
+          <v-form ref="form" v-model="valid">
+            <v-layout wrap class="ma-2">
+              <v-flex xs12 md12>
                 <v-text-field :disabled="locked" v-model="editedItem.name" label="Name"></v-text-field>
               </v-flex>
-              <v-flex xs12>
+              <v-flex xs12 md12>
                 <v-text-field :disabled="locked" v-model="editedItem.address" label="Address"></v-text-field>
               </v-flex>
-              <v-flex xs12>
-                <v-text-field v-model="editedItem.allowance" label="Value"></v-text-field>
+              <v-flex xs12 md12>
+                <v-text-field
+                  v-model="editedItem.allowance"
+                  type="text"
```

Comment:
Maybe you can use type number:
```suggestion
                  type="number"
```

Reply:
Done.